snap/snapcraft.yaml

```diff
@@ -1,4 +1,5 @@
 name: bitcoin
+base: core18
```

Comment:
I guess this is the line that enables multipass. But since it is also changing the base image from core16, I think the PR title and commit subject from #10: "snap: Bump to core18 (bionic)" would be more descriptive then the current title "Multipass" which doesn't mention core18 or snaps.

Reply:
I have updated the title to be more specific